CHANGELOG.md

```diff
@@ -1,5 +1,133 @@
+  * `conda_build.conda_interface.Resolve` (use `conda.resolve.Resolve` instead). (#5333)
+  * `conda_build.conda_interface.rm_rf` (use `conda_build.utils.rm_rf` instead). (#5333)
+  * `conda_build.conda_interface.root_dir` (use `conda.base.context.context.root_prefix` instead). (#5333)
+  * `conda_build.conda_interface.root_writable` (use `conda.base.context.context.root_writable` instead). (#5333)
+  * `conda_build.conda_interface.spec_from_line` (use `conda.cli.common.spec_from_line` instead). (#5333)
+  * `conda_build.conda_interface.specs_from_args` (use `conda.cli.common.specs_from_args` instead). (#5333)
+  * `conda_build.conda_interface.specs_from_url` (use `conda.cli.common.specs_from_url` instead). (#5333)
+  * `conda_build.conda_interface.StringIO` (use `io.StringIO` instead). (#5333)
+  * `conda_build.conda_interface.subdir` (use `conda.base.context.context.subdir` instead). (#5333)
+  * `conda_build.conda_interface.symlink_conda`. (#5333)
+  * `conda_build.conda_interface.TemporaryDirectory` (use `conda.gateways.disk.create.TemporaryDirectory` instead). (#5333)
+  * `conda_build.conda_interface.TmpDownload` (use `conda.gateways.connection.download.TmpDownload` instead). (#5333)
+  * `conda_build.conda_interface._toposort` (use `conda.common.toposort._toposort` instead). (#5333)
+  * `conda_build.conda_interface.unix_path_to_win` (use `conda.utils.unix_path_to_win` instead). (#5333)
+  * `conda_build.conda_interface.untracked` (use `conda.misc.untracked` instead). (#5333)
+  * `conda_build.conda_interface.Unsatisfiable` (use `conda.exceptions.UnsatisfiableError` instead). (#5333)
+  * `conda_build.conda_interface.UnsatisfiableError` (use `conda.exceptions.UnsatisfiableError` instead). (#5333)
+  * `conda_build.conda_interface.url_path` (use `conda.utils.url_path` instead). (#5333)
+  * `conda_build.conda_interface.VersionOrder` (use `conda.models.version.VersionOrder` instead). (#5333)
+  * `conda_build.conda_interface.walk_prefix` (use `conda.misc.walk_prefix` instead). (#5333)
+  * `conda_build.conda_interface.win_path_to_unix` (use `conda.common.path.win_path_to_unix` instead). (#5333)
+  * `conda_build.index.channel_data`; `conda_build.index.get_build_index` return value for `channel_data` is now always `None`. (#5333)
+  * `conda_build.metadata.check_circular_dependencies` (use `conda_build.metadata._check_circular_dependencies` instead). (#5406)
+  * `conda_build.metadata.toposort` (use `conda_build.metadata.toposort_outputs` instead). (#5406)
+  * `conda_build.utils._convert_lists_to_sets` (use `frozendict.deepfreeze` instead). (#5333)
+  * `conda_build.utils.HashableDict` (use `frozendict.deepfreeze` instead). (#5333)
+  * `conda_build.utils.represent_hashabledict` (use `frozendict.deepfreeze` instead). (#5333)
+  * `conda_build.utils.rm_rf(config)`. (#5333)
+  * `conda_build.variants.get_vars(loop_only)`. (#5333)
+
+### Other
+
+* Long `conda info` error messages for common exceptions. (#5264)
```

Comment:
Long? Log?

Reply:
```suggestion
* Log `conda info` error messages for common exceptions. (#5264)
```
🪵

Reply:
```suggestion
* Do not generate conda error reports for common exceptions. (#5264)
```

Reply:
Oops, good catch! I cut + pasted the issue title, which is what got fixed (the error reports were too long)